app/src/androidTest/java/org/mozilla/fenix/ui/robots/SearchRobot.kt

```diff
@@ -187,7 +187,7 @@ class SearchRobot {
         onView(withContentDescription(expectedText))
     }
     fun verifyDefaultSearchEngine(expectedText: String) = assertDefaultSearchEngine(expectedText)
-    fun verifyFillLinkButton() = assertFillLinkButton()
+    fun verifyFillLinkFromClipboardIsNotDisplayed() = assertFillLinkFromClipboardIsNotDisplayed()
```

Comment:
Could you move this next to https://github.com/mozilla-mobile/fenix/blob/main/app/src/androidTest/java/org/mozilla/fenix/ui/robots/NavigationToolbarRobot.kt#L68 so we can have them both in one place, for a better visibility?

Reply:
Done !